Features3/tests/features/deleteContact.feature

```diff
@@ -0,0 +1,29 @@
+Feature: Delete a contact
+    As a user 
+    I want to delete contact 
+    So that contact will be removed from contact list
+
+    Background:
+        Given the User has logged in and navigated to Contact Fox page
+
+    Scenario Outline: Delete all created contacts from contact list
+        Given Contacts are added with name as "<name>", email as "<email>", phone as "<phone>" and  contact type as "<contact-type>"
+        When User deletes a contact with name "<name>", email "<email>", phone "<phone>" and contact type "<contact-type>"
+        Then that contact with name "<name>", email "<emai>", phone "<phone>" and contact type "<contact-type>" should be removed from contact list 
```

Comment:
i think we need not to provide all data to delete contact. we can check uniquely created email and delete it something like:
```
When User deletes a contact with email "<email>"
Then User with email "<email>" should be deleted
```

like you have done below.

Reply:
corrected by passing email value only